### DISCUSSION TITLE
Adaptive sens speed

### DISCUSSION THREAD

**Discussion by DarkGradient** [ppy/osu#25387] [Ideas]
My idea looks something like this:
The user enters two desired and comfortable sense values (like 1.65 and 1.85 ), and then osu will try to automatically adjust it for the best game. (Only available after 10 seconds of gameplay, as with offset.)
But I think it might look like "Cheating"
( Sorry if this topic is duplicated, I just didn't find anything suitable :) )

**Comment by bdach** [comment]
How would this even... work? Conceptually? Is the game supposed to guess what the player was intending to do?

**Reply by DarkGradient** [reply]
Not really, but rather the game should "dynamically" change the cursor speed during the game.
And the change itself should occur based on two values ​​​​set by the player.
For example: If a player starts to fall short somewhere, then the game will try to increase the value, but if on the contrary, then lower it.
